### PR TITLE
Merge Spring Spring Cloud Azure AppConfiguration 2.10.0 to main

### PR DESCRIPTION
**Pull request by Netyyyy**
As title. After AppConfiguration release, need to merge to main branch.

### CODE REVIEW

File: sdk/appconfiguration/azure-spring-cloud-appconfiguration-config-web/CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
-- Fixed an issue where Feature Flags didn't NormalizeNull on refresh checks, causing `(No Label)`/empty label feature flags to not cause refreshes when their value changed.
-
 ### Other Changes
 
+## 2.10.0 (2022-11-24)
+- This release is compatible with Spring Boot 2.5.0-2.5.14, 2.6.0-2.6.13, 2.7.0-2.7.5. (Note: 2.5.x (x>14), 2.6.y (y>13) and 2.7.z (z>5) should be supported, but they aren't tested with this release.)
+- This release is compatible with Spring Cloud 2020.0.3-2020.0.6, 2021.0.0-2021.0.5. (Note: 2020.0.x (x>6) and 2021.0.y (y>5) should be supported, but they aren't tested with this release.)
+
+### Bugs Fixed
+- Fix CVEs.
```
Comment (chenrujun) [inline sdk/appconfiguration/azure-spring-cloud-appconfiguration-config-web/CHANGELOG.md:18]: Could you please add more information about `CVEs` here?

File: sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/CHANGELOG.md [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
-- Fixed an issue where Feature Flags didn't NormalizeNull on refresh checks, causing `(No Label)`/empty label feature flags to not cause refreshes when their value changed.
-
 ### Other Changes
 
+## 2.10.0 (2022-11-24)
+- This release is compatible with Spring Boot 2.5.0-2.5.14, 2.6.0-2.6.13, 2.7.0-2.7.5. (Note: 2.5.x (x>14), 2.6.y (y>13) and 2.7.z (z>5) should be supported, but they aren't tested with this release.)
+- This release is compatible with Spring Cloud 2020.0.3-2020.0.6, 2021.0.0-2021.0.5. (Note: 2020.0.x (x>6) and 2021.0.y (y>5) should be supported, but they aren't tested with this release.)
+
+### Bugs Fixed
+- Fix CVEs.
```
Comment (chenrujun) [inline sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/CHANGELOG.md:18]: Same here.

File: sdk/appconfiguration/azure-spring-cloud-starter-appconfiguration-config/CHANGELOG.md [hunk 3]
```diff
@@ -10,6 +10,13 @@
 
 ### Other Changes
 
+## 2.10.0 (2022-11-24)
+- This release is compatible with Spring Boot 2.5.0-2.5.14, 2.6.0-2.6.13, 2.7.0-2.7.5. (Note: 2.5.x (x>14), 2.6.y (y>13) and 2.7.z (z>5) should be supported, but they aren't tested with this release.)
+- This release is compatible with Spring Cloud 2020.0.3-2020.0.6, 2021.0.0-2021.0.5. (Note: 2020.0.x (x>6) and 2021.0.y (y>5) should be supported, but they aren't tested with this release.)
+
+### Bugs Fixed
+- Fix CVEs.
```
Comment (chenrujun) [inline sdk/appconfiguration/azure-spring-cloud-starter-appconfiguration-config/CHANGELOG.md:18]: Same here.

File: sdk/appconfiguration/azure-spring-cloud-feature-management/CHANGELOG.md [hunk 4]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+## 2.9.0 (2022-11-24)
+- This release is compatible with Spring Boot 2.5.0-2.5.14, 2.6.0-2.6.13, 2.7.0-2.7.5. (Note: 2.5.x (x>14), 2.6.y (y>13) and 2.7.z (z>5) should be supported, but they aren't tested with this release.)
+- This release is compatible with Spring Cloud 2020.0.3-2020.0.6, 2021.0.0-2021.0.5. (Note: 2020.0.x (x>6) and 2021.0.y (y>5) should be supported, but they aren't tested with this release.)
```
Comment (chenrujun) [inline sdk/appconfiguration/azure-spring-cloud-feature-management/CHANGELOG.md:15]: No `CVE` here?

**Review by chenrujun — CHANGES_REQUESTED** [review]

**Review by saragluna — APPROVED** [review]
LGTM.

**Comment by saragluna** [comment]
@mssfang could you help us with the changelog, for we are not sure which CVEs are addressed in this release?

**Comment by mssfang** [comment]
> @mssfang could you help us with the changelog? For we are not sure which CVEs are addressed in this release.

The newest Azure core bumps the external dependency `woodstox-core` up to date that fixed the below list of vulnerabilities, fixed in 6.4.0.
For example, [GHSA-fv22-xp26-mm9w](https://github.com/advisories/GHSA-fv22-xp26-mm9w)

NAME                     INSTALLED  FIXED-IN  TYPE          VULNERABILITY        SEVERITY
woodstox-core            6.3.1      6.4.0     java-archive  GHSA-3f7h-mf4q-vrm4  Low
woodstox-core            6.3.1      6.4.0     java-archive  GHSA-fv22-xp26-mm9w  High
woodstox-core            6.3.1      6.4.0     java-archive  GHSA-4rv7-wj6m-6c6r  Low
woodstox-core            6.3.1      6.4.0     java-archive  GHSA-9fwf-46g9-45rx  Low
woodstox-core            6.3.1      6.4.0     java-archive  GHSA-5hc5-c3m9-8vcj  Low

**Comment by saragluna** [comment]
Thanks @mssfang. @Netyyyy please update the changelog according to Shawn's comment.

File: sdk/appconfiguration/azure-spring-cloud-feature-management/CHANGELOG.md (hunk 4 above)
Comment (Netyyyy) [reply sdk/appconfiguration/azure-spring-cloud-feature-management/CHANGELOG.md:15]: They're not depend on `azure-core`.

**Comment by Netyyyy** [comment]
Since these CVEs have been marked "REJECT" in the CVE List, will ignore these in CHANGELOG. 
woodstox-core            6.3.1      6.4.0     java-archive  GHSA-4rv7-wj6m-6c6r  Low
woodstox-core            6.3.1      6.4.0     java-archive  GHSA-9fwf-46g9-45rx  Low
woodstox-core            6.3.1      6.4.0     java-archive  GHSA-5hc5-c3m9-8vcj  Low

**Review by chenrujun — APPROVED** [review]

**Pull request merged**